ghost/members-api/test/unit/lib/repositories/member.test.js

```diff
@@ -439,5 +545,56 @@ describe('MemberRepository', function () {
             memberModelStub.add.calledOnce.should.be.true();
             memberModelStub.add.args[0][0].newsletters.should.eql([]);
         });
+
+        it('does not subscribe a member to an archived newsletter, from a list of active newsletters', async function () {
```

Comment:
I don't think we need this test, the same behaviour is already covered in `does not subscribe a member to an archived newsletter`